simulation.js

```diff
@@ -2,16 +2,17 @@ class Simulation {
 	constructor() {
 		this.particles = [];
 
-		this.AMOUNT_PARTICLES = 2000;
+		this.AMOUNT_PARTICLES = 1200;
 		this.VELOCITY_DAMPING = 1;
```

Comment:
is 2000 particles too high for a meaningful fps?

Reply:
for my laptop, it was when on battery power

Reply:
This wont be a problem now that we have particle emitters.